openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion.py

```diff
@@ -177,6 +177,18 @@ def author_view(self, context=None):  # pylint: disable=unused-argument
         """
         Renders author view for Studio.
         """
+        return self.studio_view_fragment()
+
+    def preview_view(self, context=None):  # pylint: disable=unused-argument
+        """
+        Renders preview inside Studio.
+        """
+        return self.studio_view_fragment()
+
+    def studio_view_fragment(self):
+        """
+        Returns a fragment to view it in studio
```

Comment:
@ziafazal Nit: Change this to "Returns a fragment for rendering this block in Studio.", perhaps?